spec/spec_helper.rb

```diff
@@ -1,5 +1,5 @@
-$:.unshift File.expand_path("../../lib", __FILE__)
-$:.unshift File.expand_path("../..", __FILE__)
+$:.unshift File.expand_path("../lib", __dir__)
+$:.unshift File.expand_path("..", __dir__)
```

Comment:
I'm curious why the root directory would need to be in LOAD_PATH 🤔

Reply:
That's a great question